Compilation error

@ghaerr , after the recent changes in memset I got the following compilation error after doing `make clean`:
```
In file included from /Users/x2241064/Install/8088/ELKS/elks/elks/include/linuxmt/fs.h:17:0,
                 from /Users/x2241064/Install/8088/ELKS/elks/elks/include/linuxmt/sched.h:7,
                 from buffer.c:4:
buffer.c: In function ‘zero_buffer’:
/Users/x2241064/Install/8088/ELKS/elks/elks/include/linuxmt/memory.h:75:25: error: too few arguments to function ‘fmemsetb’
 #define xms_fmemset     fmemsetb
                         ^
buffer.c:650:9: note: in expansion of macro ‘xms_fmemset’
         xms_fmemset((char *)offset, ebh->b_L2seg, count);
         ^~~~~~~~~~~
/Users/x2241064/Install/8088/ELKS/elks/elks/include/linuxmt/memory.h:19:6: note: declared here
 void fmemsetb (void * off, seg_t seg, byte_t val, size_t count);
      ^~~~~~~~
make[3]: *** [buffer.o] Error 1
make[2]: *** [fs/fs.a] Error 2
make[1]: *** [Image] Error 2
make: *** [all] Error 2

```

now if I remove 3rd argument in `void fmemsetb (void * off, seg_t seg, byte_t val, size_t count);` in `memory.h` compilation brakes later. After that, if I restore `memory.h` compilation completes successfully,

--- a/elks/include/linuxmt/memory.h
+++ b/elks/include/linuxmt/memory.h
@@ -70,9 +70,9 @@ void linear32_fmemset(void *dst_off, addr_t dst_seg, byte_t val, size_t count);
 #else
 
 typedef seg_t ramdesc_t;	/* ramdesc_t is just a regular segment descriptor */
-#define xms_fmemcpyw	fmemcpyw
-#define xms_fmemcpyb	fmemcpyb
-#define xms_fmemset     fmemsetb
+#define xms_fmemcpyw	            fmemcpyw
+#define xms_fmemcpyb	            fmemcpyb
+#define xms_fmemset(off,seg,cnt)    fmemsetb(off, seg, 0, cnt)
 
 #endif /* CONFIG_FS_XMS */
 

